<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="0ms"
		durationms="0"
		type="root"
		rid="88482c66f8d2b98"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesscount="0"
		totalfailedcount="1"
		totalblockedcount="0"
		user="Karun Shrestha"
		host="THINK-PK2N901"
		rxversion="6.2.0.0"
		osversion="Windows 7 Service Pack 1 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		language="en-US"
		screenresolution="1600x900"
		timestamp="1/14/2017 1:44:57 PM"
		timestampiso="2017-01-14T18:44:57"
		timeoutfactor="1"
		module="CreateUserNeg">
		<detail>
			CreateUserNeg
		</detail>
		<varbindings/>
		<item
			timeRelativeToTestSuiteStartTime="00:10.150"
			timeRelativeToTestModuleStartTime="00:10.150"
			timeWallClock="13:45:07"
			time="00:10.150"
			level="Error"
			category="User"
			errimg=".\CreateUs_b2784334_24789900_trace.jpg"
			errthumb=".\CreateUs_b2784334_24789900_trace_thumb.jpg">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error"/>
		</item>
	</activity>
</report>
